src/contracts/polygon/AaveV3PolMaticXRiskParamsUpdatePayload.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import {AaveV3Polygon, AaveV3PolygonAssets} from 'aave-address-book/AaveV3Polygon.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+
+/**
+ * @title Update Risk Params for MaticX on Aave V3 Polygon
+ * @author Llama
+ * @dev This proposal updates multiple risk parameters for the MaticX pool on Aave V3 Polygon
+ * Governance: https://governance.aave.com/t/arfc-maticx-polygon-v3-upgrade/11555
+ * Snapshot: https://snapshot.org/#/aave.eth/proposal/0x8abe6a2ae1134d6886f460d6648dc3a5d2f789e1b94e78f496e7f5ec5ff38697
+ */
+contract AaveV3PolMaticXRiskParamsUpdatePayload is IProposalGenericExecutor {
+  uint256 public constant NEW_SUPPLY_CAP = 8_600_000;
+  uint256 public constant NEW_BORROW_CAP = 5_200_000;
+  uint256 public constant NEW_LIQ_FEE = 10_00;
+  uint256 public constant NEW_LTV = 58_00;
+  uint256 public constant NEW_LIQ_THRESHOLD = 67_00;
+  uint256 public constant NEW_LIQ_BONUS = 1_10_00;
+  address public constant NEW_INTEREST_RATE_STRATEGY = 0x6B434652E4C4e3e972f9F267982F05ae0fcc24b6;
+
+  function execute() external {
```

Comment:
I saw in the snapshot that the reserver factor has to be set to 20% as well. Does that not need to be updated or is the current reserve factor already at 20%?

Reply:
It's already 20: https://github.com/bgd-labs/aave-proposals/pull/66/files#diff-3b9324de337d75929c89476ab5304251c242c43cb0c92da012024420bed4f021R11